### PR TITLE
[minor] Add custom labels support to Tekton

### PR DESCRIPTION
**Pull request by racree**
Pass custom labels environment variable to ansible devops code.

### CODE REVIEW

File: tekton/pipeline-templates/taskdefs/common/cli-params.yml.j2 [hunk 1]
```diff
@@ -4,6 +4,8 @@
 # Parameters that enable MongoDb results storage
 - name: mas_instance_id
   value: $(params.mas_instance_id)
+- name: custom_labels
```
Comment (durera) [inline tekton/pipeline-templates/taskdefs/common/cli-params.yml.j2]: We can only params here if the param is used in every taskdef.  We will need to add custom_labels to each of the files in `tekton/pipeline-templates/*/*yml.j2` that match with the 12 files the parameter was added to in `tekton/tasks/*.yaml` , e.g. `tekton/pipeline-templates/dependencies/appconnect.yml.j2`

**Review by durera — CHANGES_REQUESTED** [review]

File: tekton/pipeline-templates/taskdefs/common/cli-params.yml.j2 (hunk 1 above)
Comment (racree) [reply tekton/pipeline-templates/taskdefs/common/cli-params.yml.j2]: The tasks with "custom_labels" are 
tekton/tasks/mongodb.yaml
tekton/tasks/suite-install.yaml
tekton/tasks/kafka.yaml
tekton/tasks/uds.yaml
tekton/tasks/suite-app-install.yaml
tekton/tasks/db2.yaml
tekton/tasks/suite-dns.yaml
tekton/tasks/gencfg-workspace.yaml
tekton/tasks/appconnect.yaml
tekton/tasks/sls.yaml
tekton/tasks/cos.yaml
tekton/tasks/suite-app-config.yaml

As requested, I removed "custom_labels" from common/cli-params.yml.j2 
and added "custom_labels" to the following:
tekton/pipeline-templates/taskdefs//core/gencfg-workspace.yml.j2
tekton/pipeline-templates/taskdefs//core/suite-install.yml.j2
tekton/pipeline-templates/taskdefs//core/suite-dns.yml.j2
tekton/pipeline-templates/taskdefs//dependencies/db2.yml.j2
tekton/pipeline-templates/taskdefs//dependencies/appconnect.yml.j2
tekton/pipeline-templates/taskdefs//dependencies/cos.yml.j2
tekton/pipeline-templates/taskdefs//dependencies/uds.yml.j2
tekton/pipeline-templates/taskdefs//dependencies/mongo.yml.j2
tekton/pipeline-templates/taskdefs//dependencies/kafka.yml.j2
tekton/pipeline-templates/taskdefs//dependencies/sls.yml.j2
tekton/pipeline-templates/taskdefs//apps/manage-app.yml.j2
tekton/pipeline-templates/taskdefs//apps/safety-workspace.yml.j2
tekton/pipeline-templates/taskdefs//apps/manage-workspace.yml.j2
tekton/pipeline-templates/taskdefs//apps/predict-app.yml.j2
tekton/pipeline-templates/taskdefs//apps/predict-workspace.yml.j2
tekton/pipeline-templates/taskdefs//apps/iot-app.yml.j2
tekton/pipeline-templates/taskdefs//apps/monitor-workspace.yml.j2
tekton/pipeline-templates/taskdefs//apps/assist-workspace.yml.j2
tekton/pipeline-templates/taskdefs//apps/hputilities-workspace.yml.j2
tekton/pipeline-templates/taskdefs//apps/safety-app.yml.j2
tekton/pipeline-templates/taskdefs//apps/optimizer-app.yml.j2
tekton/pipeline-templates/taskdefs//apps/optimizer-workspace.yml.j2
tekton/pipeline-templates/taskdefs//apps/hputilities-app.yml.j2
tekton/pipeline-templates/taskdefs//apps/visualinspection-app.yml.j2
tekton/pipeline-templates/taskdefs//apps/visualinspection-workspace.yml.j2
tekton/pipeline-templates/taskdefs//apps/assist-app.yml.j2
tekton/pipeline-templates/taskdefs//apps/monitor-app.yml.j2
tekton/pipeline-templates/taskdefs//apps/iot-workspace.yml.j2

**Comment by racree** [comment]
fvt1
fvt2
fvt3

Verified changes do not break fvt pipeline scripts

**Comment by racree** [comment]
fvt5
fvt4

verified CUSTOM_LABELS environment variable set in .travis.yml is set on deployed resources.

**Review by durera — APPROVED** [review]

**Pull request merged**